### PR TITLE
TT-3374 - SDK ALGO - wip

### PR DESCRIPTION
**Pull request by isra67**

### CODE REVIEW

File: examples/algo-example/src/index.ts [hunk 1]
```diff
@@ -15,27 +15,36 @@ import { SDKArguments } from '@tatumio/shared-abstract-sdk'
 
 export const sdkArguments: SDKArguments = {
   apiKey: '75ea3138-d0a1-47df-932e-acb3ee807dab',
+  url: 'http://127.0.0.1:3000',
   // url: 'https://api.tatum.io',
+  provider: '',
```
Comment (samuelsramko) [inline examples/algo-example/src/index.ts:20]: Delete this line. On above line, use https://api.Tatum.io/v3/blockchain/node/ALGO

**Review by samuelsramko — APPROVED** [review]

File: examples/algo-example/src/app/algo.asa-transfer.example.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  const { address, secret } = algoSDK.wallet.generateWallet()
+  const fromPrivateKey = secret
+  const recipientAddress = algoSDK.wallet.generateWallet()
+  const to = recipientAddress.address
+
+  // FUND YOUR ACCOUNT WITH ALGOs FROM https://bank.testnet.algorand.network/
+
+  // Send Algos to a recipient account using private key, to be able accept token
+  // https://apidoc.tatum.io/tag/Algorand#operation/AlgorandBlockchainTransfer
+  const txData = (await algoSDK.transaction.signedTransaction(
```
Comment (Hathoriel) [inline examples/algo-example/src/app/algo.asa-transfer.example.ts:25]: This algoSDK.transaction.signedTransaction is meant to be like this? I though it should be encapsulated like this:
algoSDK.transaction.send.signedTransaction

File: examples/algo-example/src/index.ts [hunk 3]
```diff
@@ -4,38 +4,50 @@
...
+
 const examples = async () => {
   console.log(`\nRunning algoBalanceExample`)
   await algoBalanceExample()
-  console.log(`\nRunning algoBlockchainExample`)
-  await algoBlockchainExample()
-  // @TODO: allow after release new version
-  // console.log(`\nRunning algoNftExample`)
-  // await algoNftExample()
+  // console.log(`\nRunning algoBlockchainExample`)
```
Comment (Hathoriel) [inline examples/algo-example/src/index.ts:32]: Dont forget to uncomment all these.

File: packages/api-client/src/generated/models/ChainBurnErc20KMS.ts [hunk 4]
```diff
@@ -6,7 +6,7 @@ export type ChainBurnErc20KMS = {
     /**
      * Chain to work with.
      */
-    chain: 'ETH' | 'BSC' | 'MATIC' | 'KLAY' | 'XDC' | 'ONE' | 'ALGO';
```
Comment (Hathoriel) [inline packages/api-client/src/generated/models/ChainBurnErc20KMS.ts:9]: Why we are removing algo here?

Comment (isra67) [reply packages/api-client/src/generated/models/ChainBurnErc20KMS.ts:9]: ALGO has its own model

File: examples/algo-example/src/app/algo.nft-burn.example.ts [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  const fromPrivateKey = secret
+
+  // FUND YOUR ACCOUNT WITH ALGOs FROM https://bank.testnet.algorand.network/
+
+  // upload your file to the ipfs:
+  // https://docs.tatum.io/guides/blockchain/how-to-store-metadata-to-ipfs-and-include-it-in-an-nft
+
+  // Mint NFTs on your own smart contract
+  // https://apidoc.tatum.io/tag/NFT-(ERC-721-or-compatible)#operation/NftMintErc721
+  const nftMinted = (await algoSDK.token.nft.createNFTSignedTransaction(
```
Comment (Hathoriel) [inline examples/algo-example/src/app/algo.nft-burn.example.ts:20]: add prepare/send

**Pull request merged**